locale/de/LC_MESSAGES/django.po

```diff
@@ -903,7 +910,7 @@ msgstr "Fehler"
 
 #: liqd_product/templates/email_base.html:47
 msgid "_LIQD_PRODUCT_ is a participation platform operated by"
-msgstr ""
+msgstr "_LIQD_PRODUCT_ ist eine Plattform für Beteiligung, betrieben durch"
```

Comment:
_LIQD_PRODUCT_ has been replaced in other places.

Reply:
not yet. the PR has not been merged. it has been only one place where we still used meinBerlin instead of _LIQD_PRODUCT_